README.md

```diff
@@ -18,6 +18,8 @@ This is a community project so feel free to contribute ideas, submit bugs, fix b
 
 To use the GCM, you can download the [latest installer](<https://github.com/Microsoft/Git-Credential-Manager-for-Windows/releases/latest>). To install, double-click Setup.exe and follow the instructions presented.
 
+When prompted to select your terminal emulator for Git Bash you should choose the Windows' default console window. GCM will not prompt you for credentials in a MinTTY setup.
```

Comment:
It's more of a "will not" versus "cannot" statement. There's no way for the GCM, that I know of, to grab the std pipes from MinTTY and therefore it **cannot** prompt for credentials when using the MinTTY command line. It however **can** prompt for credentials when using the MinTTY command line if the user sets `credential.modalpropmpt true`.